CHANGELOG.md

```diff
@@ -16,50 +16,51 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 - **Headline** via [#PR_NUMBER](https://github.com/apollographql/router/pull/PR_NUMBER)
 
-  Description! And a link to a [reference]
-
-  [reference]: http://link
+  Description! And a link to a [reference](http://url)
 
  -->
  
-# Upcoming
-
-> Not yet in a release!
+# [v0.1.0-alpha.7] 2022-02-25
 
 ## :sparkles: Features
 
-- **Apollo Studio Explorer landing page** ([PR #526](https://github.com/apollographql/router/pull/526)
+- **Apollo Studio Explorer landing page** ([PR #526](https://github.com/apollographql/router/pull/526))
 
   We've replaced the _redirect_ to Apollo Studio with a statically rendered landing page.  This supersedes the previous redirect approach was merely introduced as a short-cut.  The experience now duplicates the user-experience which exists in Apollo Gateway today.
 
+  It is also possible to _save_ the redirect preference and make the behavior sticky for future visits.  As a bonus, this also resolves the failure to preserve the correct HTTP scheme (e.g., `https://`) in the event that the Apollo Router was operating behind a TLS-terminating proxy, since the redirect is now handled client-side.
+
+  Overall, this should be a more durable and more transparent experience for the user.
+
 - **Display Apollo Router version on startup** ([PR #543](https://github.com/apollographql/router/pull/543))
   The Apollo Router displays its version on startup from now on, which will come in handy when debugging/observing how your application behaves.
 
 ## :bug: Fixes
 
-  It is also possible to _save_ the redirect preference and make the behavior sticky for future visits.  As a bonus, this also resolves the failure to preserve the correct HTTP scheme (e.g., `https://`) in the event that the Apollo Router was operating behind a TLS-terminating proxy, since the redirect is now handled client-side.
+- **Passing a `--supergraph` file supersedes Managed Federation** ([PR #535](https://github.com/apollographql/router/pull/535))
 
-  Overall, this should be a more durable and more transparent experience for the user.
+  The `--supergraph` flag will no longer be silently ignored when the Supergraph is already being provided through [Managed Federation](https://www.apollographql.com/docs/federation/managed-federation/overview) (i.e., when the `APOLLO_KEY` and `APOLLO_GRAPH_REF` environment variables are set).  This allows temporarily overriding the Supergraph schema that is fetched from Apollo Studio's Uplink endpoint, while still reporting metrics to Apollo Studio reporting ingress.
 
 - **Anonymous operation names are now empty in tracing** ([PR #525](https://github.com/apollograpqhl/router/pull/525))
 
```

Comment:
My practice of testing markdown links is paying off.  My practice of manually typing URLs is not.
```suggestion
- **Anonymous operation names are now empty in tracing** ([PR #525](https://github.com/apollographql/router/pull/525))

```